packages/ui-extensions/src/surfaces/checkout/components/QRCode/QRCode.doc.ts

```diff
@@ -42,10 +42,9 @@ const data: ReferenceEntityTemplateSchema = {
       anchorLink: 'best-practices',
       title: 'Best Practices',
       sectionContent: `
-        - Always test that the QR code is scannable from a smartphone.
-        - Include a square logo if that’s what your customers are familiar with.
-        - Increase usability by adding a text description of what the QR code does and provide alternate methods to get to the same destination, like a text input containing the value.
-      `,
+- Always test that the QR code is scannable from a smartphone.
+- Include a square logo if that’s what your customers are familiar with.
+- Increase usability by adding a text description of what the QR code does and provide alternate methods to get to the same destination, like a text input containing the value.      `,
```

Comment:
```suggestion
- Increase usability by adding a text description of what the QR code does and provide alternate methods to get to the same destination, like a text input containing the value.`,
```
? not sure tbh haha but felt a lot of space there